corehq/apps/export/views.py

```diff
@@ -1831,11 +1820,10 @@ def get_app_data_drilldown_values(self, in_data):
         try:
             rmi_helper = ApplicationDataRMIHelper(self.domain, self.request.couch_user)
             response = rmi_helper.get_case_rmi_response()
-        except Exception as e:
+        except Exception:
             return format_angular_error(
-                _("Problem getting Create Export Form: {}").format(e.message),
+                _("Problem getting Create Export Form"),
                 log_error=True,
-                exception=e,
```

Comment:
@snopoke what I said here about `logging.exception` is true, and it does result in an error being sent to Sentry, is that all correct?